packages/sdk/src/actor/actor.ts

```diff
@@ -458,40 +471,45 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	 * If omitted, a best-guess size is chosen based on the currently assigned mesh
 	 * (loading meshes are not considered). If no mesh is assigned, defaults to (1, 1, 1).
 	 * @param center The center of the collider, or default of the object if none is provided.
+	 * @param layer Controls what the assigned actors will collide with.
 	 */
 	public setCollider(
 		colliderType: ColliderType.Capsule,
 		isTrigger: boolean,
 		size?: Vector3Like,
-		center?: Vector3Like
+		center?: Vector3Like,
+		layer?: CollisionLayer,
 	): void;
 
 	/**
 	 * Adds a collider whose shape is determined by the current mesh.
 	 * @param colliderType Type of the collider to enable.
 	 * @param isTrigger Whether the collider is a trigger volume or not.
+	 * @param size Ignored for ColliderType Auto.
+	 * @param center Ignored for ColliderType Auto.
+	 * @param layer Controls what the assigned actors will collide with.
 	 */
 	public setCollider(
 		colliderType: ColliderType.Auto,
-		isTrigger: boolean
+		isTrigger: boolean,
+		size?: Vector3Like,
+		center?: Vector3Like,
```

Comment:
These parameters do not make sense at an API level for a collider of type auto.  I am guessing this is to support the change of moving the collider geometry determination to the _setCollider function?  Please see my question there about the reasons you are wanting to make this change.